subnet/kube/kube.go

```diff
@@ -280,6 +280,10 @@ func (ksm *kubeSubnetManager) AcquireLease(ctx context.Context, attrs *subnet.Le
 			return nil, err
 		}
 	}
+	err = ksm.setNodeNetworkUnavailableFalse()
```

Comment:
Shouldn't this be in an exponential retry loop?
Or do you want to rely on kubelet re-spinning the pod?

Reply:
I don't know realy. I suppose we will never catch the error except if the node doesn't exist anymore. 
Flanneld is restart if kublet restart ?

Reply:
@rajatchopra it's ok for you ? it's a fix for custom k8s cluster on GCE